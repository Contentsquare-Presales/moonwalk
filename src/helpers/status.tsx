const STATUS_MESSAGES = [
  "Unknown Status",
  "Launch is GO",
  "Launch is NO-GO",
  "Launch was a success",
  "Launch failed",
];

export const getStatusMessage = (statusNumber: number) =>
  STATUS_MESSAGES[statusNumber] || "Unknown Status";
