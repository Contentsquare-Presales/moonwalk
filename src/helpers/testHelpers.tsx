import React, { ReactElement } from "react";
import renderer from "react-test-renderer";
import { ThemeProvider } from "styled-components/native";

import theme from "../theme";

export const renderWithTheme = (component: ReactElement) =>
  renderer.create(<ThemeProvider theme={theme}>{component}</ThemeProvider>);