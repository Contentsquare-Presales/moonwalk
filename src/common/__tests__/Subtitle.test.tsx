import React from "react";
import { Subtitle } from "../";
import { renderWithTheme } from '../../helpers/testHelpers'

describe("Subtitle", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(<Subtitle text="SAMPLE_SUBTITLE" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
