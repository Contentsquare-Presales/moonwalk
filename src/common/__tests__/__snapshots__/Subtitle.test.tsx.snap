// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subtitle renders correctly 1`] = `
<View
  style={
    Array [
      Object {},
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#828c94",
          "fontSize": 20,
        },
      ]
    }
  >
    SAMPLE_SUBTITLE
  </Text>
</View>
`;
